src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -37,6 +38,12 @@ public GenotypeCalculationArgumentCollection( final GenotypeCalculationArgumentC
     }
 
     /**
+     * Use the new allele frequency / QUAL score model
+     */
+    @Argument(fullName = "useNewAFCalculator", shortName = "newQual", doc = "If provided, we will use the new AF model instead of the so-called exact model", optional = true)
+    public boolean USE_NEW_AF_CALCULATOR = false;
```

Comment:
Yes, I have been testing in `HaplotypeCaller` i.e. in the GATK4 integration tests.

So right now the toggle for the new qual model is in `GenotypeCalculationArgumentCollection` while the `AFCalculatorImplementation` switch `pnrm` is still in `StandardCallerArgumentCollection`.  This is not unified and is in fact quite clumsy but it seems to me that the alternative is to make the new model fit into the whole apparatus of `AFCalculatorImplementation`, `AFCalculatorProvider` etc.  Since this is all going to be torn down pretty soon I personally prefer the temporary clumsiness over enmeshing the new model in code that will soon be deleted.

If you see things differently let me know and I will put everything under `pnrm`.